testing/test_terminal.py

```diff
@@ -2138,3 +2138,24 @@ def test_foo():
                 ]
             )
         )
+
+    def test_code_highlight_false(self, testdir: Testdir, color_mapping) -> None:
+        testdir.makepyfile(
+            """
+            def test_foo():
+                print('''
+                '''); assert 0
+        """
+        )
+        result = testdir.runpytest("--code-highlight=no")
+        color_mapping.requires_ordered_markup(result)
+        result.stdout.fnmatch_lines(
+            color_mapping.format_for_fnmatch(
+                [
+                    "    {kw}def{hl-reset} {function}test_foo{hl-reset}():",
+                    "        {print}print{hl-reset}({str}'''{hl-reset}{str}{hl-reset}",
+                    ">   {str}    {hl-reset}{str}'''{hl-reset}); {kw}assert{hl-reset} {number}0{hl-reset}",
+                    "assert 0{reset}",
+                ]
```

Comment:
I believe the test does not work, I'm not familiar with this format. I shall fix it.